Dockerfile-linux.template

```diff
@@ -180,10 +186,10 @@ RUN set -eux; \
 		--enable-shared \
 {{
 	# <3.10 does not have -fno-semantic-interposition enabled and --with-lto does nothing for performance
-	# skip LTO on alpine on riscv64: https://github.com/docker-library/python/pull/935, https://github.com/docker-library/python/pull/1038
+	# skip LTO on riscv64: https://github.com/docker-library/python/pull/935, https://github.com/docker-library/python/pull/1038
 	if rcVersion == "3.9" then "" else (
 -}}
-		$(test "$gnuArch" != 'riscv64-linux-musl' && echo '--with-lto') \
+		$(test "${gnuArch%%-}" != 'riscv64' && echo '--with-lto') \
```

Comment:
this `%%` syntax doens't seem to work for me, `$(if ! echo "${gnuArch%%-}" | grep -q 'riscv64'; then echo '--with-lto' ; fi)` does, but maybe there's a nicer way...

Reply:
that works, thanks 👍🏿